src/Plugin.php

```diff
@@ -70,9 +70,11 @@ public static function post_gallery($output = '', $atts) {
     $atts['post_type'] = 'attachment';
     $atts['post__in'] = $atts['include'];
     $layout = !empty($atts['layout']) ? $atts['layout'] : apply_filters('gallerya/default_layout', self::DEFAULT_LAYOUT);
+    $nav_count_min = apply_filters('gallerya/nav_count_min', 6);
     ob_start();
     static::renderTemplate(['templates/layout-' . $layout . '.php'], [
       'images' => get_posts($atts),
+      'nav_count_min' => $nav_count_min,
```

Comment:
Do we need this local variable or can we directly create/retrieve + filter the value when passing the parameters?